swap_meet/decor.py

```diff
@@ -1,2 +1,10 @@
-class Decor:
-    pass
+from swap_meet.item import Item 
+
+class Decor(Item):
+
+    def __init__(self, condition = 0):
+        self.category = "Decor"
+        self.condition = condition 
```

Comment:
same as above! we could refactor this a little more to use inheritance more explicitly